### DISCUSSION TITLE
How to send argument to Get.bottomsheet?

### DISCUSSION THREAD

**Discussion by Thiyraash96** [jonataslaw/getx#1012] [Q&A]
First of all thanks for the wonderful package. I have question. How do we send arguments to widget which we open using Get.bottomsheet(); Normal we can send argument from one page to another using Get.toNamed('/path',argument:data). is there anything similar for the Get.bottomsheet(Widget(), argument:data)

**Comment by eduardoflorence** [comment]
Hi @Thiyraash96, you can send via the Widget constructor:
`Get.bottomsheet(Widget(arguments: data))`

**Comment by Thiyraash96** [comment]
Thanks @eduardoflorence for the suggestion. Yes we can send the data using Widget's constructor. Is this the only way we can send data to Get.BottomSheet 's widget? We does not have anything like Get.bottomsheet(Widget(), argument:data)

**Comment by qu-Ming** [comment]
Try this @Thiyraash96 
Get.bottomsheet(Widget,  settings: RouteSettings(arguments: [abc]),)
